.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
we don't need to figure this out in this PR something for us to think about...it might be better if we can be more descriptive for tags so we can easily distinguish between snapshots vs full releases just by the tag name